documentation/examples/prometheus-mesos-cluster.yml

```diff
@@ -0,0 +1,50 @@
+global:
+  scrape_interval:     15s # By default, scrape targets every 15 seconds.
+
+  # Attach these labels to any time series or alerts when communicating with
+  # external systems (federation, remote storage, Alertmanager).
+  external_labels:
+    monitor: 'mesos_cluster'
+
+# A scrape configuration containing exactly one endpoint to scrape:
+# Here it's Prometheus itself.
+scrape_configs:
+  - job_name: 'mesos_nodes'
+    # If you have mesos dns configured on the machine running 
+    # prometheus you can use dns to find all nodes to pull from
+    dns_sd_configs:
+    - names:
+      - slave.mesos
+      type: 'A'
+      # Port to pull from
+      # Default port for node exporter
+      port: 9100
+  
+  # Find the master nodes
+  - job_name: 'mesos_masters'
```

Comment:
This is actually plural in the case of multiple masters.  Mesos dns will return multiple values.
http://mesosphere.github.io/mesos-dns/docs/naming.html